.github/workflows/infrastructure.yml

```diff
@@ -1,76 +1,267 @@
+            --query peeringState)"
+
+      - name: Web app - UI 🌐
+        uses: Azure/cli@v1.0.7
+        with:
+          inlineScript: |
+            az webapp create \
+            --name app-ui-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}.azurewebsites.net \
+            --plan ${{ env.TARGET }} \
+            --deployment-container-image-name cr${{ env.PRODUCT }}${{ env.TARGET }}${{ vars.VERSION }}.azurecr.io/ui:${{ env.TARGET }} \
+            --vnet vnet-${{ env.PRODUCT }}-${{ vars.VERSION }} \
+            --tags ${{ env.TAGS }}
+
+      - name: Web app - API 🌐
+        uses: Azure/cli@v1.0.7
+        with:
+          inlineScript: |
+            az webapp create \
+            --name app-api-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}.azurewebsites.net \
+            --plan ${{ env.TARGET }} \
+            --deployment-container-image-name cr${{ env.PRODUCT }}${{ env.TARGET }}${{ vars.VERSION }}.azurecr.io/api:${{ env.TARGET }} \
+            --vnet vnet-${{ env.PRODUCT }}-${{ vars.VERSION }} \
+            --tags ${{ env.TAGS }}
+
+  # 3. Web app configuration
+  webapp:
+    name: Web app configuration 🔧
+    needs: base
+    environment: ${{ needs.base.outputs.environment }}
+    runs-on: [self-hosted, EXIP, infrastructure]
+    steps:
+      - name: Pre-production 💫
+        if: contains('["dev", "staging"]', env.TARGET)
+        run: echo "TYPE=Preproduction" >> $GITHUB_ENV
+
+      - name: Production 💫
+        if: ${{ 'prod' == env.TARGET }}
+        run: echo "TYPE=Production" >> $GITHUB_ENV
+
+      - name: Login 🔐
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Azure defaults ✨
+        uses: Azure/cli@v1.0.7
+        with:
+          inlineScript: |
+            # Basic
+            az configure --defaults location=${{ vars.REGION }}
+            az configure --defaults group=rg-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}
+
+      - name: CD ✨
+        run: |
+          # Configure continuous deployment via containers
+
+          # UI
+          az webapp deployment container config \
+          --name app-ui-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}.azurewebsites.net \
           --enable-cd true
 
-          az webapp config set --name ${{ env.product }}-${{ env.plan }}-${{ env.name }} \
-          --always-on true
+          # API
+          az webapp deployment container config \
+          --name app-api-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}.azurewebsites.net \
+          --enable-cd true
+
+        - name: Configuration ✨
+        run: |
+          # Set a web app's configuration
+
+          # UI
+          az webapp config set \
+          --name app-ui-${{ env.PRODUCT }}-${{ env.TARGET }}-${{ vars.VERSION }}.azurewebsites.net \
+          --always-on true \
+          --ftps-state Disabled \
+          --http20-enabled true \
+          --min-tls-version '1.2'
```

Comment:
I've seen the tls version repeating a few times. Is it worth having it in a var?

Reply:
Have introduced `TLS_VERSION`.